corehq/apps/users/views/utils.py

```diff
@@ -115,58 +116,84 @@ def log_commcare_user_locations_changes(request, user, old_location_id, old_assi
         )
 
 
-def _get_location_ids_with_single_user(domain, location_ids, user_id):
-    # Remove the user's ID, as we want to see which other CommCare users share any of the given location_ids
-    user_query = (
+def _flatten_list(list_2d):
+    flat_list = []
+    # Iterate through the outer list
+    for element in list_2d:
+        if isinstance(element, list):
+            # If the element is of type list, iterate through the sublist
+            for item in element:
+                flat_list.append(item)
+        else:
+            flat_list.append(element)
+    return flat_list
+
+
+def _get_location_ids_with_other_users(domain, location_ids, user_id):
```

Comment:
Since `user_id` is excluded, I think a proper name is in order for the parameter. Something like `exclude_user_id` or even `exclude_user_ids` if we expect a list